test/budgie_web/live/budget_show_live_test.exs

```diff
@@ -50,7 +65,7 @@ defmodule BudgieWeb.BudgetShowLiveTest do
       conn = log_in_user(conn, user)
       {:ok, lv, _html} = live(conn, ~p"/budgets/#{budget}/new-transaction")
 
-      assert has_element?(lv, "#create-transaction-modal")
```

Comment:
`create-transaction-modal` was renamed to `transaction-modal` to be more generic